### PR TITLE
T23 - Voting contest

### PR DESCRIPTION
**Pull request by karthick-1205**

### CODE REVIEW

File: MainRepository/Program.cs [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+   else {
+      VotingContest (input, out int totalVotes);
+      Console.WriteLine ($"The total votes is {totalVotes}");
+   }
+}
+
+static void VotingContest (string s, out int votes) {
+   Dictionary<char, int> voteCount = new ();
+   foreach (char c in s) {
+      var result = voteCount.ContainsKey (c) ? voteCount[c]++ : voteCount[c] = 1;
```
Comment (priyarathna) [inline MainRepository/Program.cs]: Why you are using this way instead of if-else?

File: MainRepository/Program.cs [hunk 2]
```diff
@@ -1,7 +1,20 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿while (true) {
```
Comment (priyarathna) [inline MainRepository/Program.cs:1]: There is now way to exit the program. Maybe if the user types 'q'/'Q', you can exit it.

File: MainRepository/Program.cs (hunk 1 above)
Comment (priyarathna) [reply MainRepository/Program.cs]: @sdinesh-90 : Which one would be better? Please advise.

Comment (sdinesh-90) [reply MainRepository/Program.cs]: As there is assignment, for clarity, it is better to use if else loop

**Review by priyarathna — APPROVED** [review]

**Pull request closed without merging**